source/includes/version/_10-show.md.erb

```diff
@@ -0,0 +1,59 @@
+## Get version
+
+```shell
+$ curl 'https://ci.example.com/go/api/version' \
+    -i \
+    -H 'Accept: application/vnd.go.cd.v1+json'
+```
+
+> The above command returns JSON structured like this:
+
+```http
+HTTP/1.1 200 OK
+Date: Wed, 01 Jun 2016 15:41:49 GMT
+Expires: Thu, 01 Jan 1970 00:00:00 GMT
+X-Frame-Options: SAMEORIGIN
+X-XSS-Protection: 1; mode=block
+X-Content-Type-Options: nosniff
+X-UA-Compatible: chrome=1
+Content-Type: application/vnd.go.cd.v1+json; charset=utf-8
+Cache-Control: max-age=0, private, must-revalidate
+X-Request-Id: 9beae519-3ff6-4cc8-98c6-047c8747fc21
+X-Runtime: 0.023000
+Vary: Accept-Encoding,User-Agent
+ETag: "4161b5022521b01ecf469ce82e23cad9"
+Set-Cookie: JSESSIONID=q58j58usvkp35ratgksl6klk;Path=/go;Expires=Wed, 15-Jun-2016 15:41:49 GMT;Secure
+Via: 1.1 build.go.cd
+Strict-Transport-Security: max-age=31536000; includeSubDomains; preload
+Transfer-Encoding: chunked
```

Comment:
The headers that are rendered are minimal. We normally render just 2 things, the response code, and the `Content-Type`, the rest of them are un-necessary noise.

```
HTTP/1.1 200 OK
Content-Type: application/vnd.go.cd.v2+json; charset=utf-8
```